script-template

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
can we eliminate this script?

Reply:
Probably, but I'd like to take care of that in a follow up.